wof/core.py

```diff
@@ -49,31 +52,33 @@
 curr_folder = os.path.abspath(os.path.dirname(__file__))
 
 _SERVICE_PARAMS = {
-    "r_type" : "rest",
-    "s_type" : "soap",
-    "wml10_tns" : "http://www.cuahsi.org/his/1.0/ws/",
-    "wml11_tns" : "http://www.cuahsi.org/his/1.1/ws/",
-    "wml10_rest_name" : "WaterOneFlow_rest_1_0",
-    "wml10_soap_name" : "WaterOneFlow_soap_1_0",
-    "wml11_rest_name" : "WaterOneFlow_rest_1_1",
-    "wml11_soap_name" : "WaterOneFlow_soap_1_1",
+    "r_type": "rest",
+    "s_type": "soap",
+    "wml10_tns": "http://www.cuahsi.org/his/1.0/ws/",
+    "wml11_tns": "http://www.cuahsi.org/his/1.1/ws/",
+    "wml10_rest_name": "WaterOneFlow_rest_1_0",
+    "wml10_soap_name": "WaterOneFlow_soap_1_0",
+    "wml11_rest_name": "WaterOneFlow_rest_1_1",
+    "wml11_soap_name": "WaterOneFlow_soap_1_1",
 }
-#utc_time_zone = tz(None,0)
-UTC_TZ=pytz.utc
+# utc_time_zone = tz(None,0)
+UTC_TZ = pytz.utc
+
 
 def site_map(app):
     output = []
     for rule in app.url_map.iter_rules():
-        methods = ','.join(rule.methods)
-        #line = urllib.unquote("{:50s} {:20s} {}".format(rule.endpoint, methods, rule))
+        # methods = ','.join(rule.methods)
```

Comment:
It is note used so I commented it. Actually there are plenty of unused commented out code already. We should remove as much as we can IMO.